pedalboard/ExternalPlugin.h

```diff
@@ -543,8 +544,8 @@ template <typename ExternalPluginType> class ExternalPlugin : public Plugin {
     }
   }
 
-  void
-  process(const juce::dsp::ProcessContextReplacing<float> &context) override {
+  int process(
```

Comment:
Any thoughts on making this an [int64_t](https://en.cppreference.com/w/cpp/types/integer) or maybe an unsigned long? Looks like unsigned at the min as negative return shouldn't be possible